### PR TITLE
Support for Pushshift switchover to COLO (as of 2022-12)

### PR DESCRIPTION
**Pull request by Dominyk4s**
Just a quick temporary fix
TL;DR:
Added two parameters: _sort_var_ and _check_totals_ to API calls; add variables `sort_var='order', check_totals=False` while calling `search_submissions` or any other function for it to work with the latest data in Pushshift

Longer version:

After moving to COLO (related Reddit post is [here](https://www.reddit.com/r/pushshift/comments/zkggt0/update_on_colo_switchover_bug_fixes_reindexing/)), The current wrapper stopped working.
Known differences:
* _order_ instead of _sort_ parameter to sort data in the response (added parameter sort_var, defaults to older _sort_, but can be replaced to _order_ in API call functions)
* the metadata (including the total number of items in the database) is not returned anymore, therefore added a parameter _check_totals_, where _False_ value will skip checking it (can cause a discrepancy in data)

### CODE REVIEW

**Comment by mattpodolak** [comment]
hey @Dominyk4s, thanks for opening this PR! I'll review it today/tomorrow

**Comment by mattpodolak** [comment]
@Dominyk4s there was a similar PR that addressed a couple of the changes in the COLO switchover, can you review the conflicts?

File: pmaw/PushshiftAPI.py [hunk 1]
```diff
@@ -32,6 +34,8 @@ def search_submission_comment_ids(self, ids, **kwargs):
             mem_safe (boolean, optional) - If True, stores responses in cache during operation, defaults to False
             safe_exit (boolean, optional) - If True, will safely exit if interrupted by storing current responses and requests in the cache. Will also load previous requests / responses if found in cache, defaults to False
             cache_dir (str, optional) - An absolute or relative folder path to cache responses in when mem_safe or safe_exit is enabled
+            sort_var (str, optional) - parameter, used in Pushshift API, to sort values
+            check_totals (bool, optional) - if to check total counts before returning results. API stopped returning total counts as of 2022-12-17
```
Comment (mattpodolak) [inline pmaw/PushshiftAPI.py:38]: I was reading the reddit post linked, it looks like we can add `&track_total_hits=true` to ensure the total counts are now accurate

**Comment by Dominyk4s** [comment]
> @Dominyk4s there was a similar PR that addressed a couple of the changes in the COLO switchover, can you review the conflicts?

@mattpodolak, I reviewed another PR, which seems to solve the same thing, just with a different approach (changes the defaults there and I'm adding everything as variables here). And I see you moved forward with even more updates, so we can close this PR 😊
